packages/safe-core-sdk-types/src/ethereumLibs/EthAdapter.ts

```diff
@@ -13,6 +13,8 @@ export interface EthAdapterTransaction {
   value?: string
   gasPrice?: number
   gasLimit?: number
+  maxFeePerGas?: number | string
+  maxPriorityFeePerGas?: number | string
```

Comment:
Is there a reason we support `string` here but not for `gasPrice`?

Reply:
Given the regular values that `gasPrice` takes, `number` is enough. However, it should be a `string | number` like the properties that were added in this PR.
I will recheck all the types of the similar properties and update them in a new PR